internal/cmd/config/profile/import/import.go

```diff
@@ -0,0 +1,106 @@
+package _import
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/config"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+)
+
+const (
+	nameFlag   = "name"
+	configFlag = "config"
+	noSetFlag  = "no-set"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	ProfileName string
+	Config      string
+	NoSet       bool
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import",
+		Short: "Imports a CLI configuration profile",
+		Long:  "Imports a CLI configuration profile.",
+		Example: examples.Build(
+			examples.NewExample(
+				`Import a config with name "PROFILE_NAME" from file "./config.json"`,
+				"$ stackit config profile --name PROFILE_NAME --config `@./config.json`",
+			),
+			examples.NewExample(
+				`Import a config with name "PROFILE_NAME" from file "./config.json" and set not as active`,
```

Comment:
'set not as active' sounds not so understandable

maybe we can say `set as inactive` or 'do not set as active'